Accessibility: add Overview topic (#5042)
DevExpress/devextreme-documentation: concepts/Common/Accessibility/00 Overview/35 Standard Compliance.md

> Compliance with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards is essential for DevExtreme components to prioritize accessibility.

Reviewer: Compliance is essential for DevExtreme components to prioritize accessibility? Can you explain what you mean here? Do you mean DevExtreme components prioritize accessibility?

Author: Yes, I mean that. But I kinda meant that DevExtreme components need to be compliant with these standards, and this is the thing that we need to do when we prioritize accessibility.

Reviewer: How about 

```suggestion
DevExtreme components prioritize accessibility and comply with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards.
```

or

```suggestion
Compliance with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards ensure DevExtreme components prioritize accessibility.
```

or

```suggestion
DevExtreme components prioritize accessibility by complying with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards.
```

> The following document is based on the <a href="https://www.itic.org/policy/accessibility/vpat" target="_blank">Voluntary Product Accessibility Template (VPAT)</a> and contains detailed information about the accessibility features DevExtreme supports.

Reviewer: ```suggestion
The following document is based on the <a href="https://www.itic.org/policy/accessibility/vpat" target="_blank">Voluntary Product Accessibility Template (VPAT)</a> and contains detailed information about the supported accessibility features.
```
